sphinx/config.py

```diff
@@ -169,7 +169,9 @@ class Config(object):
         epub_theme = ('epub', 'html'),
         epub_theme_options = ({}, 'html'),
         epub_title = (lambda self: self.html_title, 'html'),
+        epub_description = ('', 'epub3', [str]),
```

Comment:
Did you check it?
It seems epub3.py is not changed yet.
So these config variables would not work.

If possible, you should add tests that these variables modifies `content.opf`.
It will save you and codes.